package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "scripts": {
     "dev": "webpack serve --config webpack/webpack.dev.cjs --host 0.0.0.0 --port 3003",
-    "build": "webpack --config webpack/webpack.prod.cjs",
+    "build": "cross-env ENVIRONMENT=stg-direct webpack --config webpack/webpack.prod.cjs",
```

Comment:
`stg-comm`

Reply:
Do I need to move the stg-direct now?

Reply:
yes :D Just make sure browser or one device can launch with `stg-comm` to make sure its correct. No need to test more than this